addons/l10n_fr/data/account.account.template.csv

```diff
@@ -520,7 +437,7 @@
 "pcg_6235","Primes","623500","expense","l10n_fr.l10n_fr_pcg_chart_template","account.account_tag_operating","False"
 "pcg_6236","Catalogues et imprimés","623600","expense","l10n_fr.l10n_fr_pcg_chart_template","account.account_tag_operating","False"
 "pcg_6237","Publications","623700","expense","l10n_fr.l10n_fr_pcg_chart_template","account.account_tag_operating","False"
-"pcg_6238","Publicité, publications, relations publiques - Divers (pourboires, dons courants...)","623800","expense","l10n_fr.l10n_fr_pcg_chart_template","account.account_tag_operating","False"
+"pcg_6238","Divers (pourboires, dons courants)","623800","expense","l10n_fr.l10n_fr_pcg_chart_template","account.account_tag_investing","False"
```

Comment:
Really, we put it in investing? Isn't it weird? I know there are some gifts, but this feels like small donations and not investment